.circleci/config.yml

```diff
@@ -138,6 +138,132 @@ jobs:
                   docker push quay.io/cybozu/$image:$BRANCH
               fi
             done
+  buildx:
+    # This is a parameterized job
+    # https://circleci.com/docs/2.0/reusing-config/#authoring-parameterized-jobs
+    parameters:
+      container-image:
+        type: string
+      dir:
+        type: string
+        default: ""
+      attach:
+        type: boolean
+        default: false
+      resource-class:
+        type: string
+        default: medium
+      before:
+        type: string
+        default: ""
```

Comment:
`attach`, `resource-class`, and `before` are not used on callers' sides.
`attach` and `resource-class` are defined in `build` and were used in the first commit for MySQL, so it would be reasonable to put them here.
I'm not sure `before` is useful.